xlive/H2MOD/Modules/SpecialEvents/SpecialEvents.cpp

```diff
@@ -522,15 +522,12 @@ namespace SpecialEvents
 						placement.variant_name = 0;
 						break;
 					}
-					placement.position.x = std::get<2>(scen_place);
-					placement.position.y = std::get<3>(scen_place);
-					placement.position.z = std::get<4>(scen_place);
-					real_point3d ori;
-					ori.x = std::get<5>(scen_place);
-					ori.y = std::get<6>(scen_place);
-					ori.z = std::get<7>(scen_place);
-					set_orientation(&placement.orientation, &placement.up, &ori);
-					placement.scale = std::get<8>(scen_place);
+					placement.position.x = std::get<2>(scen_place).i;
```

Comment:
u can just placement.position = std::get<2>(scen_place);, no need to go through all components

Reply:
it places them in i,j,k instead of x,y,z when using those types.